problem_builder/mentoring.py

```diff
@@ -865,6 +881,89 @@ def steps(self):
             child_isinstance(self, child_id, MentoringStepBlock)
         ]
 
+    def get_steps(self):
```

Comment:
@itsjeyd The `questions` vs `get_questions` remark also applies here (`steps` vs `get_steps`).